docs/installation_linux.rst

```diff
@@ -1,18 +1,24 @@
 Install LISA on Linux
 =====================
 
-We will guide you through the installation of LISA on Linux.
+Minimum System Requirement
+-----------------------
+1. Your favourite Linux distro supporting Python 3.8+
+2. Dual core processor
+3. 4 GB system memory
+
+We will guide you through the installation of LISA on Linux. The following commands assume Ubuntu 18.04 as the underlying Linux distro.
 
 Install Python on Linux
 -----------------------
 
-LISA has been tested on `Python 3.8 64
-bits <https://www.python.org/>`__. The latest version of Python 3 is
-recommended. If you found LISA is not compatible with higher version
+LISA has been tested to work with `Python 3.8 64-bit
+<https://www.python.org/>`__. The latest version of Python 3 is
+recommended. If you find that LISA is not compatible with higher version
 Python, `please file an
 issue <https://github.com/microsoft/lisa/issues/new>`__.
 
-Run following commands to install Python 3.8 in Ubuntu 20.04.
+Run following commands to install Python 3.8:
```

Comment:
What's the reason to remove Ubuntu? The below commands work on Ubuntu only.

Reply:
You can explain that other distros can install the similar packages.

Reply:
We had mentioned ubuntu multiple times in the command. I removed them and instead added the line 
"We will guide you through the installation of LISA on Linux. The following commands assume Ubuntu 18.04 as the underlying Linux distro." before the commands